### DISCUSSION TITLE
Multilingual fields and built-in translation in "wagtail"

### DISCUSSION THREAD

**Discussion by paragliding-nsk** [wagtail/wagtail#11002] [Q&A] [answered]
Please tell me if I understand correctly that “wagtail” does not have built-in automatic translation for user pages (I don't mean the admin interface) and database tables? 
If so, please tell me which libraries or services for automatic translation are best suited for "wagtail"?

**Comment by lb-** [comment]
We have quite thorough documentation on this (internationalisation). 
https://docs.wagtail.org/en/stable/advanced_topics/i18n.html

**Comment by gasman** [comment, marked as answer]
If you're looking for automated machine translation in particular, see https://www.wagtail-localize.org/how-to/integrations/machine-translation/

**Reply by paragliding-nsk** [reply]
Thanks a lot
